internal/pkg/cli/app_package.go

```diff
@@ -40,11 +49,54 @@ func NewPackageAppOpts() *PackageAppOpts {
 
 // Ask prompts the user for any missing required fields.
 func (opts *PackageAppOpts) Ask() error {
+	if opts.AppName == "" {
+		names, err := opts.listAppNames()
+		if err != nil {
+			return err
+		}
+		if len(names) == 0 {
+			return errors.New("there are no applications in the workspace, run `archer init` first")
+		}
+		app, err := opts.prompt.SelectOne("Which application's CloudFormation template would you like to generate?", "", names)
```

Comment:
Nit: I tend to dislike using possessives in this way. Maybe "Which application would you like to generate a CFN template for?"?

Reply:
:+1: that is better! Updated.